randomstuff/_helper.py

```diff
@@ -1,10 +1,12 @@
 import inspect
+from colorama import init
 from .errors import *
 
 def _warn(client, warning):
     if client.suppress_warnings:
         return
     else:
+        init()
         print("\u001b[33m"+"[WARNING] "+warning)
         print("\u001b[36m"+"\n[INFO] Disable warnings by setting suppress_warnings to `True` in client." + "\u001b[0m")
```

Comment:
Is it confirmed that these will work even after `init`, What if we use `colorama.Fore.*` colors?

Reply:
Other then that, everything looks good. 👍🏻

Reply:
Seems like the `init()` function will support existing color codes. On Linux based systems this method will have no effect as they already support these codes, but on windows this will make the color codes into win32 calls giving as the colored text as intended. 
For more info
https://pypi.org/project/colorama/

Reply:
Oh great then